plugins/pencilblue/controllers/article.js

```diff
@@ -33,11 +33,9 @@ Article.prototype.render = function(cb) {
 	var custUrl = this.pathVars.customUrl;
 
 	//check for object ID as the custom URL
-	var doRedirect = false;
-	var where      = null;
+	var where  = null;
 	try {
```

Comment:
Right, but keep in mind that the bug here is that the custom URL is being validated as an ObjectID. It's a cosmetic change, more than anything.